hs_core/hydroshare/utils.py

```diff
@@ -26,7 +26,7 @@
 from hs_core.signals import pre_create_resource, post_create_resource, pre_add_files_to_resource, \
     post_add_files_to_resource
 from hs_core.models import AbstractResource, BaseResource, ResourceFile
-from hs_core.hydroshare.hs_bagit import create_bag_files
+from hs_core.hydroshare import hs_bagit
```

Comment:
It'd be better to only import the create_bag method from hs_bagit module rather than importing the whole module since this create_bag method is the only one being used in this module.